doc/mgr/orchestrator_cli.rst

```diff
@@ -223,7 +259,51 @@ Start/stop/reload::
     ceph orchestrator service-instance {start,stop,reload} <type> <instance-name>
 
 
-Removing services::
-
-    ceph orchestrator {mds,rgw} rm <name>
-
+Current Implementation Status
+=============================
+
+This is an overview of the current implementation status of the orchestrators.
+
+=================================== ========= ====== ========= =====
+ Command                             Ansible   Rook   DeepSea   SSH
+=================================== ========= ====== ========= =====
+ host add                            ⚪         ⚪       ⚪         ✔️
+ host ls                             ⚪         ⚪       ⚪         ✔️
+ host rm                             ⚪         ⚪       ⚪         ✔️
+ mgr update                          ⚪         ⚪       ⚪         ⚪
+ mon update                          ⚪         ⚪       ⚪         ⚪
+ osd create                          ✔️         ✔️       ⚪         ⚪
+ osd device {ident,fault}-{on,off}   ⚪         ⚪       ⚪         ⚪
+ osd rm                              ✔️         ⚪       ⚪         ⚪
+ device {ident,fault}-(on,off}       ⚪         ⚪       ⚪         ⚪
+ device ls                           ✔️         ✔️       ✔️         ⚪
+ service ls                          ⚪         ✔️       ✔️         ⚪
+ service status                      ⚪         ✔️       ✔️         ⚪
+ service-instance status             ⚪         ❌       ⚪         ⚪
```

Comment:
I think this could be applicable to rook, fwiw. There's nothing fundamentally wrong with, e.g., asking the status of a single container within an nfs or mds service.

Reply:
fixed